utopia-remix/app/routes/projects.tsx

```diff
@@ -647,115 +594,139 @@ const ProjectCard = React.memo(
       return renaming?.type === 'rename' ? renaming.newTitle : project.title
     }, [project, activeOperations])
 
+    const accessRequestsFetcher = useFetcher()
+    const [accessRequests, setAccessRequests] = React.useState<
+      ProjectAccessRequestWithUserDetails[]
+    >([])
+
+    React.useEffect(() => {
+      const action = `/internal/projects/${project.proj_id}/access/requests`
+      accessRequestsFetcher.submit({}, { method: 'GET', action: action })
+      if (accessRequestsFetcher.state === 'idle' && accessRequestsFetcher.data != null) {
+        if (isProjectAccessRequestWithUserDetailsArray(accessRequestsFetcher.data)) {
+          setAccessRequests(accessRequestsFetcher.data)
+        }
+      }
+    }, [accessRequestsFetcher, project])
```

Comment:
This is causing/will cause an endless loop of requests to that fetcher! 😱 
1. The effect should not submit the fetcher, but instead _only_ be used to process new data on the fetcher (lines 605-609)
2. The submit should only happen on an explicit action – I think the right place would when the context menu is opened, see https://github.com/concrete-utopia/utopia/blob/4bbd34c90173a22e7d13a1cb13641c4a64a08f44/utopia-remix/app/routes/projects.tsx#L905-L906